automation_tools/satellite6/upgrade/template.py

```diff
@@ -0,0 +1,121 @@
+from automation_tools.satellite6.hammer import hammer, set_hammer_config
+from automation_tools.satellite6.upgrade import \
+    check_necessary_env_variables_for_upgrade
+from automation_tools.satellite6.upgrade.tasks import \
+    get_rhevm_client, wait_till_rhev_instance_status
+from fabric.api import run, execute
+from ovirtsdk.xml import params
+from time import sleep
+import os
+import sys
+
+
+# Restart the Services
+def service_restart():
+    try:
+        services = run('katello-service restart')
+        if services.return_code > 0:
+            sys.exit(1)
+    except Exception as ex:
+        print ex
+        sys.exit(1)
+
+
+# Running capsule sync on external capsule
+def check_capsule():
+    set_hammer_config(user=None, password=None)
+    try:
+        capsules = hammer('capsule list')
+        cap_id = 0
+        for item in capsules:
+            if str(item[u'name']).__contains__("capsule"):
+                cap_id = str(item[u'id'])
+                print cap_id
+        check = hammer("capsule refresh-features --id {0}".format(cap_id))
+        print check[u'message']
+        if check.return_code == 0:
+            print "running capsule sync"
+    except Exception as ex:
+        print ex
+        sys.exit(1)
+
+
+# Check if ntpd is running
+def check_ntpd():
+    try:
+        ntpd_check = run("service ntpd status", warn_only=True)
+        if ntpd_check.return_code > 0:
+            run("service ntpd start")
+            run("chkconfig ntpd on")
+    except Exception as ex:
+        print ex
+        sys.exit(1)
+
+
+# Check for Vm's and create their templates
+def create_template(host, cluster, new_template, storage):
```

Comment:
This function is a little bit confusing, could you explain what you are trying to accomplish?

Having those two try/except block seems to be redundant